python/mxnet/io.py

```diff
@@ -89,20 +104,21 @@ class DataBatch(object):
     label : list of NDArray, optional
           A list of input labels.
     pad : int, optional
-          The number of examples padded at the batch end. It is used when the
-          examples read is less than the batch size.
+          The number of examples padded at the end of a batch. It is used when the
+          total number of examples read is not divisible by the `batch_size`.
+          These are ignored in the result.
```

Comment:
"These extra padded examples are ignored during processing."

Is the above a correct statement to make?